geomesa-convert/geomesa-convert-avro/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2014 Commonwealth Computer Research, Inc.
```

Comment:
The final form of the headers is still under discussion.  I think 2013-2015 makes sense as well.